src/main/java/io/stargate/sgv2/jsonapi/service/resolver/matcher/TableFilterResolver.java

```diff
@@ -0,0 +1,92 @@
+package io.stargate.sgv2.jsonapi.service.resolver.matcher;
+
+import io.stargate.sgv2.jsonapi.api.model.command.Command;
+import io.stargate.sgv2.jsonapi.api.model.command.Filterable;
+import io.stargate.sgv2.jsonapi.api.model.command.clause.filter.FilterOperation;
+import io.stargate.sgv2.jsonapi.api.model.command.clause.filter.JsonType;
+import io.stargate.sgv2.jsonapi.api.model.command.clause.filter.ValueComparisonOperator;
+import io.stargate.sgv2.jsonapi.config.OperationsConfig;
+import io.stargate.sgv2.jsonapi.service.cqldriver.executor.TableSchemaObject;
+import io.stargate.sgv2.jsonapi.service.operation.filters.DBFilterBase;
+import io.stargate.sgv2.jsonapi.service.operation.filters.table.NativeTypeTableFilter;
+import io.stargate.sgv2.jsonapi.service.operation.filters.table.NumberTableFilter;
+import io.stargate.sgv2.jsonapi.service.operation.filters.table.TextTableFilter;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.EnumSet;
+import java.util.List;
+
+public class TableFilterResolver<T extends Command & Filterable>
+    extends FilterResolver<T, TableSchemaObject> {
+
+  private static final Object DYNAMIC_TEXT_GROUP = new Object();
+  private static final Object DYNAMIC_NUMBER_GROUP = new Object();
+
+  public TableFilterResolver(OperationsConfig operationsConfig) {
+    super(operationsConfig);
+  }
+
+  @Override
+  protected FilterMatchRules<T> buildMatchRules() {
+    var matchRules = new FilterMatchRules<T>();
```

Comment:
We should make the match rules an Application Scoped as this doesn't change by find class for table.

Reply:
Also need to do the same in `CollectionFilterResolver`